include/zephyr/dt-bindings/pwm/it8xxx2_pwm.h

```diff
@@ -23,6 +23,11 @@
 #define PWM_CHANNEL_6		6
 #define PWM_CHANNEL_7		7
 
+/* Init PWM channel output level */
+#define PWM_IT8XXX2_INIT_OUTPUT_LOW	0
+#define PWM_IT8XXX2_INIT_OUTPUT_HIGH	1
+#define PWM_IT8XXX2_INIT_OUTPUT_NONE	0xff
```

Comment:
what does NONE do in this context? I'd imagine once you set the pin as PWM you'd either want it full off or full on.